Formula/fastfec.rb

```diff
@@ -35,7 +32,7 @@ def install
   end
 
   test do
-    system "#{bin}/fastfec", "--no-stdin", "13425"
+    system("curl https://docquery.fec.gov/dcdev/posted/13425.fec | #{bin}/fastfec 13425")
```

Comment:
If `curl` is no longer a dependency, will these tests still work? Is there a way to specify curl as a test-only dep?